### ISSUE TITLE
Issues with Unicode characters and --no-hidden option 🍋

### ISSUE THREAD

**Issue by ItsNickBarry** [nachoparker/dutree#8]
I have a file named `🍋.mp3`.

Running `dutree -H` yields this error:
```
thread 'main' panicked at 'byte index 1 is not a char boundary; it is inside '🍋' (bytes 0..4) of `🍋.mp3`', libcore/str/mod.rs:2238:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

The error occurs with other Unicode characters as well, but only when the `-H` option is passed.

Here's the rest of the error when I set `RUST_BACKTRACE=1`:
```
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
   1: std::sys_common::backtrace::print
   2: std::panicking::default_hook::{{closure}}
   3: std::panicking::default_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::begin_panic
   6: std::panicking::begin_panic_fmt
   7: rust_begin_unwind
   8: core::panicking::panic_fmt
   9: core::str::slice_error_fail
  10: core::str::traits::<impl core::slice::SliceIndex<str> for core::ops::range::RangeTo<usize>>::index::{{closure}}
  11: dutree::Entry::new
  12: dutree::run
  13: dutree::main
  14: std::rt::lang_start::{{closure}}
  15: std::panicking::try::do_call
  16: __rust_maybe_catch_panic
  17: std::rt::lang_start_internal
  18: main
  19: __libc_start_main
  20: _start
```

**Comment by ItsNickBarry** [comment]
I have not tested this with all Unicode characters.

**Comment by nachoparker** [comment]
fixed in `v0.2.9`, thanks!
